js/customscripts.js

```diff
@@ -71,20 +71,20 @@
 
 
         //hijack clicks on links, verify if anchor link and jump to div
```

Comment:
Probably not. Just didn't have time to see what else in the theme might be depending on it. I'll test around and make sure everything's fine and delete this, if so.